src/api_server/swagger/firecracker.yaml

```diff
@@ -577,6 +653,13 @@ definitions:
         description: Flag for enabling/disabling Hyperthreading
       cpu_template:
         $ref: "#/definitions/CpuTemplate"
+      track_dirty_pages:
+        type: boolean
+        description:
+          Enable dirty page tracking. If this is enabled, then incremental guest memory
+          snapshots can be created. These are called diff snapshots and they are saving
+          in a sparse file the memory dirtied since the last snapshot, which can be a
+          full or diff one. A full snapshot contains a full copy of the guest memory.
```

Comment:
nit:
```suggestion
          snapshots can be created. These are called diff snapshots containing only
          the memory dirtied since a previous snapshot. Full snapshots each contain a
          full copy of the guest memory.
```